docs/introduction/flows.md

```diff
@@ -57,27 +57,26 @@ All flows use an action property to designate what the flow is doing and how it
 * vf:accept - in processes like repair or maintentance, the same resource will appear in output with *vf:modify* verb
 * vf:modify - in processes like repair or maintentance, the same resource will appear in input with *vf:accept* verb
 * vf:service - new service produced and delivered (being a service implies that an agent actively receives the service
-* vf:give - give rights and responsibilities for the resource
-* vf:receive - receive rights and responsibilites for the resource
+* vf:transfer - give rights and/or possession of a resource from one agent to another
+* vf:move - change location and/or identity of a resource with no change of agent
 * vf:raise - adjusts a quantity up based on a beginning balance or inventory count
 * vf:lower - adjusts a quantity down based on a beginning balance or inventory count
 
-Action | Affect | I/O | Changes existence | Pairs with |
------- | ------ | --- | ----------------- | ---------- |
-produce | Increment  | Output | Yes | N/A |
-consume | Decrement  | Input | Yes | N/A |
-use | No effect  | Input | No |N/A |
-work | No effect  | Input | N/A | N/A |
-cite | No effect  | Input | No | N/A |
-load | Decrement  | Input | No | unload |
-unload | Increment  | Output | No | load |
-accept | No effect  | Input | No | modify |
-modify | No effect  | Output | No | accept |
-service | No effect | either | Yes | N/A |
-give | Decrement | Input | No | receive |
-receive | Increment | Output | No | give |
-raise | Increment | N/A | N/A | N/A |
-lower | Decrement | N/A | N/A | N/A |
+Action | Affect | I/O | Changes existence | Changes possession | Changes availability | Pairs with |
+------ | ------ | --- | ----------------- | ------------------ | ---------------------| ---------- |
+produce | Increment  | Output | Yes | N/A | Yes | N/A |
+consume | Decrement  | Input | Yes | N/A | Yes | N/A |
+use | No effect  | Input | No | No | Yes | N/A |
+work | No effect  | Input | N/A | N/A | Yes | N/A |
+cite | No effect  | Input | No | No | No | N/A |
+load | No effect  | Input | No | N/A | Yes | unload |
+unload | No effect  | Output | No | N/A | Yes | load |
+accept | No effect  | Input | No | N/A | Yes | improve |
+improve | No effect  | Output | No | N/A | Yes | accept |
+transfer | N/A | N/A | No | N/A | N/A | N/A |
```

Comment:
Suggest removing possession and availability columns until issues resolved, and than
transfer | Decrement+Increment | N/A | No | N/A |
or
transfer | Decr.+Incr. | N/A | No | N/A |